src/vtol.py

```diff
@@ -135,21 +137,41 @@ def takeoff(self):
 
         print("Reached target altitude")
 
+    def set_altitude(self, alt):
+        '''Sets altitude of quadcopter using an "alt" parameter'''
+        print("Setting altitude:")
+        destination = LocationGlobalRelative(self.location.global_relative_frame.lat, self.location.global_relative_frame.lon, alt)
+        self.simple_goto(destination)
+        while abs(self.location.global_relative_frame.alt - alt) > 0.5:
```

Comment:
Lets move this .5 to the configs, this should be used globally as our margin of error